schemas/+aod/trackMotion3D.m

```diff
@@ -0,0 +1,86 @@
+
+function [x,y,z,t] = trackMotion3D(fn, analysischan) 
+
+%% test function computeshifts
+% requires z:\libraries\matlab in path
+run(getLocalPath('/lab/libraries/hdf5matlab/setPath.m')) ;
+
+downsampleFactor = 1 ; % skip frames if > 1
+numFramesToAverageForRef = 10 ; % frames to average to determine the reference frame against which motion is computed
+useMultiCore = true ;
+postRefStartFrameTime = 0 ; % N secs to skip before analyzing motion signals, in many cases, right after starting the point scan, there is a slight drift
+
+br = aodReader(fn, 'Motion') ;
```

Comment:
I ended up replacing this with

    fn2 = findFile(RawPathMap, fn)
    br = aodReader(fn2, 'Motion');
    br1 = AodMotionCorrectionReader(fn2) ; % if motion was corrected by using aod offsets, this is where they would be read